### PR TITLE
Add small edits and fix typos

### PR DESCRIPTION
**Pull request by willingc**
@Lukasa I intended to fix the missing space in "thank you", but wound up going through the file and doing a light edit. Just let me know what changes you would like :sunny:

### CODE REVIEW

File: source/contributing.rst [hunk 1]
```diff
@@ -1,28 +1,34 @@
 Contributing
 ============
 
-If you’re reading this you’re probably interested in contributing to one of the
-Hyper projects! First, we'd like to say thankyou! Projects like this one
+If you’re reading this, you’re probably interested in contributing to one of the
```
Comment (LilyFirefly) [inline source/contributing.rst:4]: I'm not sure the extra comma helps, but if others think it does that's fine by me,

File: source/contributing.rst [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+       - documentation
+       - bug reports
+       - feature requests
 
 Code Of Conduct
 ---------------
 
 The following code of conduct applies to all projects that are under the
-umbrella of the hyper project.
+hyper project umbrella.
```
Comment (LilyFirefly) [inline source/contributing.rst]: I prefer the old text in this case.

File: source/contributing.rst [hunk 3]
```diff
@@ -167,22 +173,22 @@ report using the guidelines in the :ref:`bug-reports` section.
 
 If you've done this but want to get contributing right away, you can take
 advantage of the fact that all Hyper projects use a continuous integration
-system. This will automatically run the tests against any pull request raised
-against any Hyper repository. The continuous integration system treats a
-regression in code coverage as a failure of the test suite.
+system. This system will automatically run the tests against any pull request
+raised against any Hyper repository. The continuous integration system treats
+a regression in code coverage as a failure of the test suite.
```
Comment (LilyFirefly) [inline source/contributing.rst:178]: The extra "system" seems unnecessary to me, but again happy if others disagree.

File: source/contributing.rst [hunk 4]
```diff
@@ -167,22 +173,22 @@ report using the guidelines in the :ref:`bug-reports` section.
...
 
-Before a contribution is merged it must have a green run through the CI system.
+Before a contribution is merged, it must have a green run through the CI system.
```
Comment (LilyFirefly) [inline source/contributing.rst:180]: Again I'm not sure the comma helps, but don't mind much.

File: source/contributing.rst [hunk 5]
```diff
@@ -1,28 +1,34 @@
...
+Hyper projects! First, we'd like to say thank you! Projects like this one
 live-and-die based on the support they receive from others, and the fact that
 you’re even considering supporting the Hyper project is incredibly generous of
 you.
 
 This document lays out general guidelines and advice for contributing to any of
-the Hyper projects. If you’re thinking of contributing, start by reading this
-thoroughly and getting a feel for how contributing to the project works. If
+the Hyper projects. If you’re thinking of helping, start by reading this
+thoroughly and getting a feel for how the project works. If
```
Comment (LilyFirefly) [inline source/contributing.rst:12]: :+1:

File: source/contributing.rst [hunk 6]
```diff
@@ -1,28 +1,34 @@
...
+We've organized this guide into sections:
+
+- our code of conduct, which applies to all contributions of any kind
+- general guidelines for all contributors
+- specific information based on the type of contribution you’re
+  thinking of making
+       - code contributions
+       - documentation
+       - bug reports
+       - feature requests
```
Comment (LilyFirefly) [inline source/contributing.rst:25]: :+1:

File: source/contributing.rst [hunk 7]
```diff
@@ -118,9 +124,9 @@ into a contribution that is not suitable for the project.
 Contribution Suitablility
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The maintainers of the specific sub-project have the last word on whether or
-not a contribution is suitable for that project. All contributions will be
-considered, but from time to time contributions will be rejected because they
+The maintainers of a sub-project have the last word on whether or
+not a contribution is suitable for that project. We consider all contributions,
+but from time to time contributions will be rejected because they
```
Comment (LilyFirefly) [inline source/contributing.rst:129]: :+1:

File: source/contributing.rst [hunk 8]
```diff
@@ -167,22 +173,22 @@ report using the guidelines in the :ref:`bug-reports` section.
...
 
 Contributions will not be merged until they've been code reviewed. You should
-implement any code review feedback unless you strongly object to it. In the
-event that you object to the code review feedback, you should make your case
-clearly and calmly. If, after doing so, the feedback is judged to still apply,
-you must either apply the feedback or withdraw your contribution.
+implement any code review feedback unless you strongly object to it. If
+you object to the code review feedback, you should make your case
+clearly and calmly. If, after doing so, the feedback is judged to apply still,
+you must either incorporate the feedback or withdraw your contribution.
```
Comment (LilyFirefly) [inline source/contributing.rst:191]: :+1:

File: source/contributing.rst [hunk 9]
```diff
@@ -210,7 +216,7 @@ project, and the general documentation for the project as a whole live in the
 `Sphinx`_ to generate the full suite of documentation.
 
 When contributing documentation, please attempt to follow the style of the
-documentation files. This means a soft-limit of 79 characters wide in your text
+documentation files. Use a soft-limit of 79 characters wide in your text
```
Comment (LilyFirefly) [inline source/contributing.rst:219]: :+1:

**Comment by willingc** [comment]
Thanks for the speedy review @Ian-Foote. As an FYI, I use a professional grammar checker for the documentation that I do on Project Jupyter. The changes made here were based on feedback from the tool. I'm happy to revert the edits above. I went back and forth on the 'umbrella' change myself, so I'll change that back to the original. The commas and extra 'system' were recommended changes by the tool.

**Comment by alexwlchan** [comment]
@Lukasa I’d merge this, but _waves hands_ something about the way GitHub handles write permissions on protected branches.

**Review by alexwlchan — APPROVED** [review]

File: source/contributing.rst [hunk 10]
```diff
@@ -1,22 +1,28 @@
...
+We've organized this guide into sections:
+
+- our code of conduct, which applies to all contributions of any kind
+- general guidelines for all contributors
+- specific information based on the type of contribution you’re
+  thinking of making
+       - code contributions
+       - documentation
+       - bug reports
+       - feature requests
```
Comment (Lukasa) [inline source/contributing.rst:25]: This is fab. In the future we should make these internal links to the relevant sections, but that's not a reason to not merge this.

**Review by Lukasa — APPROVED** [review]
Thanks so much @willingc, this looks great! Also thanks to @Ian-Foote and @alexwlchan for review. I've fixed up the darn merge permissions too. Might need to write a tool to automatically manage this, as I seem to get it wrong a lot.

File: source/contributing.rst (hunk 4 above)
Comment (sethmlarson) [reply source/contributing.rst:180]: I would argue a comma isn't correct here.

File: source/contributing.rst (hunk 8 above)
Comment (sethmlarson) [reply source/contributing.rst:191]: I think the "feedback is judged to apply still" sentence should just be rewritten, it was confusing the first time I read it and it's still confusing to read.

**Review by sethmlarson — COMMENTED** [review]
Looks good, still have a few suggestions.

**Pull request merged**